refactor: removed unused import from the provider file

alkavats1 opened the pull request:
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: N/A


## What is the new behavior?


## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

alkavats1 commented:
Hi @AndrewKushnir please review the PR
Thanks :)

AndrewKushnir commented on packages/common/http/testing/src/provider.ts:
```diff
@@ -6,7 +6,7 @@
 
-import {HttpBackend, HttpClientModule} from '@angular/common/http';
+import {HttpBackend} from '@angular/common/http';
 import {NgModule, Provider} from '@angular/core';
```
Could you please also remove the `NgModule` symbol import (it's also unused)?
```suggestion
import {Provider} from '@angular/core';
```

alkavats1 committed:
refactor: removed the unused declaration in the import statement

ngmodule and httpclient modeule is removed from the provider.ts file from the import statement

alkavats1 commented:
Hi @AndrewKushnir, I have changed the code, according to your suggestion, please review and merge
Thanks :)

AndrewKushnir reviewed (approved):
@alkavats1 thanks for the refactoring!

AndrewKushnir commented:
**Caretaker note**: low risk change, no presubmit is required.

AndrewKushnir commented:
This PR was merged into the repository by commit 66824b77154fb0fd0a4d1ffc35976cb404ec3226.
